recipe/meta.yaml

```
@@ -690,6 +866,7 @@ outputs:
         - {{ pin_subpackage("libarrow-gandiva", exact=True) }}
         - {{ pin_subpackage("libarrow-substrait", exact=True) }}
         - {{ pin_subpackage("libparquet", exact=True) }}
+        - {{ pin_subpackage("pyarrow-core", exact=True) }}
```

I'd prefer for this to depend on 
```suggestion
        - {{ pin_subpackage("pyarrow", exact=True) }}
```
That way, it's obvious that there's no "hole" in the layering.